deploy/olm-catalog/ibm-platform-api-operator/3.6.0/ibm-platform-api-operator.v3.6.0.clusterserviceversion.yaml

```
@@ -393,3 +393,5 @@ spec:
   provider:
     name: IBM
   version: 3.6.0
+    spec:
+      replaces: ibm-platform-api-operator.v3.5.0
```

Sorry, `spec.replaces` was referring to the existing `spec` field on line 97, so should be:
```suggestion
  version: 3.6.0
  replaces: ibm-platform-api-operator.v3.5.0
```